### PR TITLE
Fix/format Cypher queries

### PR DESCRIPTION
**Pull request by m-appel**
Correct some Cypher queries and do some slight formatting (remove unused variables, add relationship directions, etc.)

### CODE REVIEW

File: src/views/iyp/AS.vue [hunk 1]
```diff
@@ -482,15 +498,18 @@ export default {
       return { cypherQuery: query, params: { asn: this.asn, rankingName: 'tranco.top1M' }, mapping, data: 'popularDomains' }
     },
     getTags() {
-      const query = 'MATCH (a:AS {asn: $asn})-[c:CATEGORIZED]-(t:Tag) return t.label as tag'
+      const query = 'MATCH (:AS {asn: $asn})-[c:CATEGORIZED]->(t:Tag) return t.label as tag'
```
Comment (romain-fontugne) [inline src/views/iyp/AS.vue:501]: ORDER BY tag ?

Comment (romain-fontugne) [reply src/views/iyp/AS.vue:501]: maybe returning the reference organization for each tag could be nice

File: src/views/iyp/Country.vue [hunk 2]
```diff
@@ -121,7 +121,12 @@ export default {
     // ases stands for autonomous systems
     getASes() {
       const query =
-        'MATCH (c:Country {country_code: $cc})-[r]-(a:AS)-[:NAME]-(n:Name) WITH c.country_code AS cc, a.asn AS asn, head(collect(DISTINCT(n.name))) AS name RETURN cc, asn, name LIMIT 100'
+        `MATCH (c:Country {country_code: $cc})<-[:COUNTRY]-(a:AS)
```
Comment (romain-fontugne) [inline src/views/iyp/Country.vue:124]: [:COUNTRY {reference_name: 'nro.delegated_stats'}] ?

**Review by romain-fontugne — CHANGES_REQUESTED** [review]
I spotted a few corrections.

I think it would be nice to systematically ORDER results (actually this could be done on the frontend which could maybe offload computation)

File: src/views/iyp/AS.vue (hunk 1 above)
Comment (m-appel) [reply src/views/iyp/AS.vue:501]: I will add ORDER BY but not the other one since Roopesh will have done this for this week I think.

**Pull request merged**